src/components/ebay-table/component.ts

```diff
@@ -11,13 +19,104 @@ export interface TableCell
     renderBody: Marko.Body;
 }
 export interface TableRow extends Omit<Marko.Input<"tr">, `on${string}`> {
+    name?: TableRowName;
+    disabled?: boolean;
     cell: Marko.RepeatableAttrTag<TableCell> | Marko.AttrTag<TableCell>[];
 }
 export interface TableInput extends Omit<Marko.Input<"div">, `on${string}`> {
     header: Marko.RepeatableAttrTag<TableHeader> | Marko.AttrTag<TableHeader>[];
+    mode?: TableMode;
     row?: Marko.RepeatableAttrTag<TableRow> | Marko.AttrTag<TableRow>[];
     density?: TableDensity;
+    selected?: TableRowStateMapping;
+    "a11y-select-all-text"?: string;
+    "a11y-select-row-text"?: string;
+    "on-select"?: (event: TableSelectEvent) => void;
 }
 export interface Input extends WithNormalizedProps<TableInput> {}
 
-export default class extends Marko.Component<Input> {}
+enum RowState {
+    SELECTED = "selected",
+    DISABLED = "disabled",
```

Comment:
Let's remove this. It's not covered in Skin. We can always add it in later if/when the use case is covered in DS and build in Skin.